inception-ui-external-search/src/main/resources/META-INF/asciidoc/user-guide/external-search-page.adoc

```diff
@@ -2,14 +2,28 @@
 = Search page
 
 If document repositories have been configured in a project, the **Search** page becomes accessible
```

Comment:
The configuration is explained in the paragraph "Document Respositories", right? Maybe add a reference to this, because at this point I asked myself where this would be explained or if I should have already done something beforehand.

Reply:
Added a reference to the paragraph "Document Repositories".